MAUI/NumericEntry/Migration.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfNumericEntry | Syncfusion 
+description: Learn about Migrating from Syncfusion Xamarin NumericTextBox to Syncfusion .NET MAUI NumericEntry control and more.
+platform: maui
+control: SfNumericEntry
+documentation: ug
+---  
+
+# Migrating from Xamarin SfNumericTextBox to .NET MAUI SfNumericEntry
```

Comment:
Migrating from Xamarin SfNumericTextBox  and SfNumericUpDown to .NET MAUI SfNumericEntry

Reply:
Add SfNumericUpDown in all places